src/pages/program/[slug]/apply.tsx

```diff
@@ -50,18 +49,16 @@ const ProgramApplyPage: Page = (_) => {
   const applicant: string | string[] = router.query.as;
   // const path: string = router.asPath;
 
-  const getApplicationType = (): ApplicationType | null => {
+  const getApplicationType = (): ApplicationType => {
     switch (applicant) {
       case "mentor":
         return ApplicationType.Mentor;
-      case "mentee":
-        return ApplicationType.Mentee;
       default:
```

Comment:
We should not assume the application type is "Mentee" if it's not "mentor". This could lead to real confusion, i.e. if someone accidentally appends a character (e.g. `as=mentors`) 

There can be other ways to avoid "flashing", including changing the loading placeholder to something that more closely matches what the page would ulitmately look like, and adding loading logic.